packages/nimble-components/src/table-column/anchor/tests/table-column-anchor.stories.ts

```diff
@@ -0,0 +1,147 @@
+    },
+    {
+        firstName: 'Milhouse',
+        lastName: 'Van Houten',
+        url: 'https://www.google.com/search?q=milhouse+van+houten'
+    },
+    {
+        firstName: 'Ned',
+        lastName: 'Flanders',
+        url: 'https://www.google.com/search?q=ned+flanders'
+    },
+    {
+        firstName: 'Maggie (no link)',
+        lastName: 'Simpson'
+    },
+    {
+        lastName: 'Simpson',
+        url: 'https://www.google.com/search?q=simpsons'
+    },
+    {
+        lastName: 'Simpson'
+    }
+];
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface AnchorColumnTableArgs extends SharedTableArgs {
+    labelFieldName: string;
+    hrefFieldName: string;
+    placeholderText: string;
+    appearance: keyof typeof AnchorAppearance;
+}
+
+const anchorColumnDescription = 'The `nimble-table-column-anchor` column is used to display string fields as links or text in the `nimble-table`. If a row provides an href for a link, that cell will display a link, otherwise it will display plain text.';
+
+export const anchorColumn: StoryObj<AnchorColumnTableArgs> = {
+    parameters: {
+        docs: {
+            description: {
+                story: anchorColumnDescription
+            }
+        }
+    },
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html<AnchorColumnTableArgs>`
+        ${usageWarning('table')}
+        <${tableTag}
+            ${ref('tableRef')}
+            data-unused="${x => x.updateData(x)}"
+        >
+            <${tableColumnAnchorTag}
+                label-field-name="${x => x.labelFieldName}"
+                href-field-name="${x => x.hrefFieldName}"
+                placeholder="${x => x.placeholderText}"
+                appearance="${x => x.appearance}"
+            >
+            First Name
+            </${tableColumnAnchorTag}>
+            <${tableColumnTextTag}
+                field-name="lastName"
+            >
+            Last Name
+            </${tableColumnTextTag}>
+        </${tableTag}>
+    `),
+    argTypes: {
+        labelFieldName: {
+            name: 'label-field-name',
+            description:
+                'Set this attribute to identify which field in the data record contains the visible text value for each cell in the column. The field values must be of type `string`. If a given row does not define a property with this name, that row will use the url as the label.',
+            options: ['firstName', 'lastName'],
+            control: { type: 'radio' }
+        },
+        hrefFieldName: {
+            name: 'href-field-name',
+            description:
+                'Set this attribute to identify which field in the data record contains the link url for each cell in the column. If the field is not defined in a particular record, that cell will be displayed as plain text instead of a link. The field values must be of type `string`.',
+            options: ['url'],
+            control: { type: 'radio' }
```

Comment:
I want it visible to document, but I don't want the user changing it, because there's not really another valid value for them to change it to. If there's a more appropriate way to do that, or you think it _should_ be editable, I'm open to alternatives.

Reply:
I question the value in showing the bound string in the docs since:
1. It can't be changed
2. We don't document the type of `TData` used in this example, so knowing that the `href-field-name` was set to the `url` property on the data model doesn't really mean much to the consumer of the docs.

Reply:
Also don't think that it should be represented via a control, just as an entry in the table like we have [started to do for events](https://github.com/ni/nimble/blob/2a0427f9ccd40f1a1a2ce5c4014df91ab4e1b55a/packages/nimble-components/src/banner/tests/banner.stories.ts#L126)

In the description if you want to mention it you could add something like `Note: In this example the href-field-name is "url" to correspond to the field of that name in the data set.`

Although if we do something like that it may be nice to do it consistently across other field name definitions. I think I lean towards that being unnecessarily verbose but I'm not against it if we make it a re-usable snippet for consistency.

Reply:
Fair enough. Removed control.